tensorflow/tools/docker/Dockerfile.devel-cpu-mkl

```diff
@@ -54,7 +54,7 @@ RUN ./configure
 RUN LD_LIBRARY_PATH=${LD_LIBRARY_PATH} \
     bazel build --config=mkl \
                 --config="opt" \
-                --copt="-march=native" \
+                --copt="-march=broadwell" \
```

Comment:
I think this may not be the best solution.
TF test environments, for example, has a lot of ivybridge and sandybridge machines. This option will disable running on those machines.

Should we lock to those instead of broadwell?

Reply:
The problem here is that Eigen has an issue with AVX512 that we haven't resolved yet. Once that issue has been fixed, we'll changed this back to native so that the container can be built on any architecture.

Reply:
Just to confirm then, we are OK with this not working on ivy bridge machines?

My understanding was once we have MKL, we can actually build the rest of the binary with as little optimizations as possible. So even disabling AVX here should not deliver a big performance hit to us?
In other words, as is we wont be able to reliably run this dockerfile in TF official CI because of our ivy bridge machines. Otherwise I am ok with the rest of the change.